plumbing/transport/file/git-receive-pack

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+# Simulates the git-upload-pack command.
+# For testing purposes only.
+# See examples/git-upload-pack.
+
+path="$(dirname $0)"
+exec go run "$path/../../../examples/git-receive-pack/main.go" "$@"
```

Comment:
if is for testing write it with the test, or something, or put in a fixture folder